### PR TITLE
Location of (0,0)

### PR DESCRIPTION
**Pull request by michal-ciechan**
Specifying (0,0) is at the top left

# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/wpf/graphics-multimedia/path-markup-syntax.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Paths are discussed in              [Shapes and Basic Drawing in WPF Overview](.
   
 <a name="pointsyntax"></a>   
 ## Point Syntax  
- Describes the x- and y-coordinates of a point.  
+ Describes the x- and y-coordinates of a point where (0,0) is the top left.
```
Comment (mairaw) [inline docs/framework/wpf/graphics-multimedia/path-markup-syntax.md]: top left -> top-left corner?

**Review by mairaw — CHANGES_REQUESTED** [review]
I like the change @michal-ciechan! I just have a suggestion to add a noun after the top left adjective,

**Comment by michal-ciechan** [comment]
@mairaw Thanks for the suggestion, definitely makes sense! Have edited commit.

**Review by mairaw — APPROVED** [review]
Thank you @michal-ciechan! Your fix will show up in the live site probably later today or tomorrow.

**Pull request merged**